server/src/main/scala/de/frosner/broccoli/instances/storage/filesystem/FileSystemInstanceStorage.scala

```diff
@@ -79,11 +82,13 @@ class FileSystemInstanceStorage(storageDirectory: File) extends InstanceStorage
   @volatile
   override def writeInstanceImpl(instance: Instance): Try[Instance] = {
     val id = instance.id
-    val file = idToFile(id)
+    val file = idToFileTemp(id)
     val printStream = Try(new PrintStream(new FileOutputStream(file)))
     val afterWrite = printStream.map(_.append(Json.toJson(instance).toString()))
-    printStream.map(_.close())
-    afterWrite.map(_ => instance)
+    val finishWrite = printStream.map(_.close())
```

Comment:
I guess we should map based on `afterWrite` to also catch if appending breaks.

Reply:
Sorry I might have been confused here. If we map it on `finishWrite` we'll not have the error when appending.

Let's quickly look at it together on Monday.

Reply:
Yes.. Sure